app/controllers/api/v1/searches_controller.rb

```diff
@@ -10,6 +10,16 @@ def show
     end
   end
 
+  def autocomplete
+    return unless params[:query]&.is_a?(String)
```

Comment:
remove this and update before action `verify_query_string` and `set_page` to include this action.

Reply:
Done this in commit `set before_action for autocomplete`.
Thank you for reviewing them.